dev.md

```diff
@@ -0,0 +1,3 @@
+# Dev notes
+
+To publish a new version, please run `bash publish.sh`.
```

Comment:
CI handles publishing and updates latest version checks in SDKs on main branch tag, avoid manual publishing

Reply:
This was in the main readme? I just moved it. So just delete?

Reply:
ah ok

Reply:
yeah